src/pages/overview/pdf-services-api/howtos/index.md

```diff
@@ -172,9 +172,6 @@ Available properties:
     milliseconds for creating an initial HTTPS connection.
 -   **socketTimeout**: Default: 10000. The maximum allowed time in
     milliseconds between two successive HTTP response packets.
--   **processingTimeout**: Default: 600000. The maximum allowed time
```

Comment:
Search  - Improving the accessibility of PDFs (Available under Early Access Program) . Autotag is GA now.